api/clients/v2/relay_client.go

```diff
@@ -248,7 +250,9 @@ func (c *relayClient) initOnceGrpcConnection(key corev2.RelayKey) error {
 			return
 		}
 		c.conns.Store(key, conn)
-		c.grpcClients.Store(key, relaygrpc.NewRelayClient(conn))
+
+		wrappedClient := newLimitedRelayClient(relaygrpc.NewRelayClient(conn), key, c.config.MaxConcurrentRequests)
```

Comment:
mini nit: var name is still `wrappedClient`, could be changed to `limitedClient`

Reply:
fixed